src/ContentScript/injectors/youtube.plugin.ts

```diff
@@ -61,7 +52,6 @@ class YoutubePlugin {
 
   private onPageLoaded() {
     if (!document.querySelector('#meta-contents')) {
-      if (this.failCounter > 10) return;
```

Comment:
After this line is removed, do we still have a max failCounter guard?

Reply:
Not sure,as I mentioned before in Discord
> onUpdated 会 ok ，另外一个 concern 若检测不到 button-container 时我还是会递归检测，检测到就停了，检测不到会一直检测。
And I'm going to use recursiveNumber instead of failCount but not sure if we should set a Maximum Limit for recursiveNumber

Reply:
Because I don't encounter the case which blocking user threads,I thought we might be able to run a recursiveNumber function behind

Reply:
If we have a recursion + mutation observber outside, does this mean we don't have to loop here? Or is this still needed?